src/index.ts

```diff
@@ -75,6 +75,18 @@ export default class NewRelicLambdaLayerPlugin {
 
   public async addLogSubscriptions() {
     const funcs = this.functions;
+    let { cloudWatchFilter = [ "NR_LAMBDA_MONITORING" ] } = this.config;
+    
+    if(cloudWatchFilter.length == 1 && cloudWatchFilter[0] == "*") {
```

Comment:
Why delete the first item here? Wouldn't it be better to iterate over the array looking for `*` and if found just set the filter pattern to ""?

Reply:
The thinking here was if you wanted all the logs, you wouldn't provide a list of filters, just the one. i will make the change as suggested.